packages/material-ui/src/Select/SelectInput.js

```diff
@@ -155,6 +155,14 @@ class SelectInput extends React.Component {
     setRef(inputRef, nodeProxy);
   };
 
+  areEqualValues = (a, b) => {
```

Comment:
This is a pure function. It should be moved outside of the class.

Reply:
Do you think this function would be useful to other components? Should we move it to a shared library or something?

Reply:
I think that we can keep it in this file.

Reply:
It's more about reducing class fields usage. Those won't be defined once in the prototype but on every class initialization (bad for perf). They are also hard to minify. I also try to use language features when necessary: This function is stateless and won't be passed around so no need to bind `this`